pepper-apis/src/main/resources/db-changes/schema/DDP-4931-activity-hide-instances.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
would have preferred to use some existing db column or config rather than add new column to track if existing instances should be hidden for study activity. 
Felt like over loading with new columns. We have study_activity.exclude_from_display but that seem to be for different purpose. 
Wondering if we can consolidate these db columns and purposes
study_activity.exclude_from_display
study_activity.hide_instances
activity_instance.is_hidden

Reply:
I do think they serve different purposes and so there's different properties, but certainly agree that we should look for opportunities to consolidate.